app/views/posts/show.html.erb

```diff
@@ -1 +1,32 @@
-<h1>Selected post for a user</h1>
\ No newline at end of file
+<h1>Selected post for a user</h1>
+<div class="container">
+  <div class="posts">
+    <div class="post_header">
+      <ul>
+        <li>
+          <h3>
+            <%= @post.title %> 
+            <span>by <%= @post.author.name %></span>
+          </h3>          
+        </li>
+        <li class="likes">
+          <span>Comments: <%= @post.comment_counter %></span>,
+          <span>Likes: <%= @post.like_counter %></span>
+        </li>
+      </ul>
+    </div>
+    <div class="post_txt">
+        <p><%= @post.text %></p>
+    </div>
+  </div>
+
+  <div class="comments">
+    <ul>
+      <% @post.recent_comments.each do |comment| %>
+        <li>
+          <%= render partial: 'partials/comment', locals: { comment: comment } %>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+</div>
```

Comment:
- [ ] According to the project wireframe, kindly ensure that all the comments are displayed on the `post_show` page. ℹ️ 

![Olufe 2](https://github.com/supreeti/blogapp/assets/103386183/3b3672a0-df67-44a4-bd81-93752aeb5ba7)

**WIREFRAME**
![Olufe 2](https://github.com/supreeti/blogapp/assets/103386183/f10ba62f-b087-4550-b465-57be872dae84)